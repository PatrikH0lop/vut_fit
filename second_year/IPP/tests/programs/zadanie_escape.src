.IPPcode18
DEFVAR GF@counter
MOVE GF@counter string@ # Inicializace prom ě nn é na pr á zdn ý ř et ě zec
# Jednoduch á iterace , dokud nebude spln ě na zadan á podm í nka
LABEL while
JUMPIFEQ end GF@counter string@aaa
WRITE string@counter\032obsahuje\032
WRITE GF@counter
WRITE string@\010
CONCAT GF@counter GF@counter string@a
JUMP while
LABEL end