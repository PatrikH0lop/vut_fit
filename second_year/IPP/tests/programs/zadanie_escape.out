counter obsahuje 



counter obsahuje 
a


counter obsahuje 
aa


